### PR TITLE
add policy renewals page for Steward

### PR DESCRIPTION
**Pull request by Schparky**
Performance is lacking when processing data on the scale of the production database, so work is yet to be done here.

image

### CODE REVIEW

File: src/helpers/sort.ts [hunk 1]
```diff
@@ -1,6 +1,3 @@
-import type { Claim } from 'data/claims'
-import type { PolicyItem } from 'data/items'
```
Comment (hobbitronics) [inline src/helpers/sort.ts:2]: I am curious what prompted replacing these with any down below.

File: src/pages/admin/renewals.svelte [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  }
+</script>
+
+<style>
+
+</style>
+
+<Page>
+  <div class="flex justify-between align-items-center">
+    <Button raised on:click={() => onUpdate()}>Update</Button>
```
Comment (hobbitronics) [inline src/pages/admin/renewals.svelte:46]:
```suggestion
    <Button raised on:click={onUpdate}>Update</Button>
```
Does this work?

File: src/pages/admin/renewals.svelte [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    <Button disabled={!reportFile?.url} on:click={() => onDownload()}>Download</Button>
```
Comment (hobbitronics) [inline src/pages/admin/renewals.svelte:47]:
```suggestion
    <Button disabled={!reportFile?.url} on:click={onDownload}>Download</Button>
```

same as above

**Review by hobbitronics — APPROVED** [review]
looks good, just a few questions.

File: src/helpers/sort.ts (hunk 1 above)
Comment (Schparky) [reply src/helpers/sort.ts:2]: I didn't want to add another type to the list. Plus, it seems to make sense that a helper like this shouldn't be limited to a list of types.

File: src/pages/admin/renewals.svelte (hunk 3 above)
Comment (Schparky) [reply src/pages/admin/renewals.svelte:47]: same as above

**Pull request merged**